client/my-sites/marketplace/pages/marketplace-product-install/index.tsx

```diff
@@ -111,9 +115,16 @@ const MarketplaceProductInstall = ( {
 		isMarketplaceProductSelector( state, pluginSlug )
 	);
 
-	const wpOrgTheme = useSelector( ( state ) => getTheme( state, 'wporg', themeSlug ) );
+	const externalTheme = useSelector( ( state ) => {
```

Comment:
I changed this to `externalTheme` since it's not necessarily a wp.org theme, but "external" might sound too much like "externally managed" aka partner themes, so open to other suggestions.

Reply:
External is a bit confusing to me, maybe something like `thirdPartyTheme` ?